src/algorithm.jl

```diff
@@ -197,7 +199,7 @@ function local_solve(m::PODNonlinearModel; presolve = false)
 
     status_pass = [:Optimal, :Suboptimal, :UserLimit, :LocalOptimal]
```

Comment:
I have an issue with :Suboptimal going to status pass after experiencing certain issues with working with OBBT and OPF problems - remind to discuss in meeting @rb004f @ccoffrin

Reply:
Here we are only constructing some initial partitions and some OBBT using the solution passed by the solver. No solution updated is performed in the POD data structure. Would this be a problem (if a dependent solver pass a problematic solution)?

Reply:
OBBT and suboptimal status don't go well. I implemented an OBBT algorithm for PowerModels.jl in https://github.com/lanl-ansi/PowerModels.jl/blob/master/src/util/obbt.jl 
I am planning to make the POD obbt algorithm similar to the PowerModels version.